.github/workflows/pypi-publish.yml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     tags:
-      - v*
+      - 'v*!rc*'
```

Comment:
Just making sure I've got it right, I think this is saying don't pick up tags with 'rc' which is short for 'release candidate'. It doesn't matter a ton, but in the spirit of discussion putting it out there :)